### ISSUE TITLE
Details about the pretrained weights

### ISSUE THREAD

**Issue by va1shn9v** [nachiket273/Vision_transformer_pytorch#2]
I have a question regarding the pretrained weights, are they randomly initialized and pretrained on imagnet-1k or are these weights pretrained on imagenet-21k and finetuned on imagenet-1k. If it is the former case what accuracy did it achieve on imagenet 1K
